actions/restore.php

```diff
@@ -17,4 +17,4 @@
 $users_approved_points = $users_points['approved'];
 $user->userpoints_points = (int) $users_approved_points;
 
-return elgg_ok_response('', elgg_echo('elggx_userpoints:restore:success', [$user->getDisplayName()]), REFERER);
+return elgg_ok_response('', elgg_echo('elggx_userpoints:restore:success', [$username]), REFERER);
```

Comment:
why changed this

Reply:
The workflow here is to enter the username of the account you want to restore the userpoints count. Therefore, I thought it would be more consistent to use the username also in the success message you get - even more so as you still had the username in the error message in the action. If you would restore the points of only one account at a time it probably wouldn't matter to use the display name in the messages. But if it happens that you might need to restore a few at once it might be a distraction if you get reported a different name in the messages than you entered in the form.